### PR TITLE
Fixed incorrect helptext ParticipationTag and ParticipationRole, fixed typos

### PR DESCRIPTION
**Pull request by dzhuang**
Fixing #744

### CODE REVIEW

File: course/models.py [hunk 1]
```diff
@@ -365,20 +365,17 @@ class ParticipationTag(models.Model):
...
             verbose_name=_("Name of participation tag"))
     shown_to_participant = models.BooleanField(default=False,
-            verbose_name=_("Shown to pariticpant"))
+            verbose_name=_("Shown to participant"))
 
     def clean(self):
         super(ParticipationTag, self).clean()
 
-        import re
-        name_valid_re = re.compile(NAME_VALID_REGEX)
```
Comment (inducer) [inline course/models.py:379]: Any other uses of `NAME_VALID_REGEX` can probably also b removed in favor of `.isidentifier()`.

File: course/models.py [hunk 2]
```diff
@@ -402,7 +399,7 @@ class ParticipationRole(models.Model):
     identifier = models.CharField(
             max_length=100, blank=False, null=False,
             help_text=_("A symbolic name for this role, used in course code. "
-            "lower_case_with_underscores, no spaces. May be any string. The "
+            "Should be a valid identifier. The "
```
Comment (inducer) [inline course/models.py:402]:
```suggestion
            "Should be a valid identifier (as defined by Python). The "
```

**Review by inducer — COMMENTED** [review]
Thanks! A few comments below.

File: course/models.py (hunk 1 above)
Comment (dzhuang) [reply course/models.py:379]: I agree.

File: course/models.py (hunk 2 above)
Comment (dzhuang) [reply course/models.py:402]: OK. I'll also add that to ptag's helptext.

File: course/models.py (hunk 1 above)
Comment (dzhuang) [reply course/models.py:379]: Done.

**Comment by dzhuang** [comment]
Some thing unexpectedly happened to AppVeyor CI, which was working fine [here](https://ci.appveyor.com/project/dzhuang/relate/builds/35947575).

**Comment by inducer** [comment]
> Something happened unexpectedly in AppVeyor CI, which was working fine [here](https://ci.appveyor.com/project/dzhuang/relate/builds/35947575).

I would prefer to stop using Appveyor altogether. Github CI has Windows VMs, can we switch to those?

**Comment by dzhuang** [comment]
> > Something happened unexpectedly in AppVeyor CI, which was working fine [here](https://ci.appveyor.com/project/dzhuang/relate/builds/35947575).
> 
> I would prefer to stop using Appveyor altogether. Github CI has Windows VMs, can we switch to those?

Sure. I agree that should be on the todo list.

**Comment by dzhuang** [comment]
Is this ready to be merged? @inducer

**Comment by inducer** [comment]
LGTM. Thanks for working on this!

**Pull request merged**